src/modules/assessments/components/AssessmentAutofillButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@mui/material';
+import { SxProps } from '@mui/system';
+import ButtonTooltipContainer from '@/components/elements/ButtonTooltipContainer';
+
+export default function AssessmentAutofillButton(props: {
+  onClick: VoidFunction;
+  sx?: SxProps;
+}) {
```

Comment:
We use the below convention for components in the project. (Not because its better than other ways of declaring components, but we just want to keep it consistent).


```
const AssessmentAutofillButton: React.FC<Props> = (props) => ...


export default AssessmentAutofillButton
```